packages/fiori/src/themes/Timeline.css

```diff
@@ -21,3 +21,20 @@
 .ui5-timeline-list-item:last-child {
 	margin-bottom: 0;
 }
+
+:host([axis-orientation="Horizontal"]) .ui5-timeline-list {
+	white-space: nowrap;
+	list-style: none;
+	margin: 0;
+	padding: 0;
+}
+
+:host([axis-orientation="Horizontal"]) .ui5-timeline-list-item {
+	display: inline-block;
+	margin-left: var(--_ui5_tl_li_margin_bottom);
+}
+
+:host([axis-orientation="Horizontal"]) .ui5-timeline-scroll-container {
+	overflow: auto;
+	width: calc(100% + 1.5rem);
```

Comment:
What this 1.5rem is for? Please document it.

Reply:
Done

Reply:
Done